src/main/java/de/tum/in/www1/artemis/service/ComplaintResponseService.java

```diff
@@ -0,0 +1,88 @@
+package de.tum.in.www1.artemis.service;
+
+import de.tum.in.www1.artemis.domain.Complaint;
+import de.tum.in.www1.artemis.domain.ComplaintResponse;
+import de.tum.in.www1.artemis.domain.User;
+import de.tum.in.www1.artemis.repository.ComplaintRepository;
+import de.tum.in.www1.artemis.repository.ComplaintResponseRepository;
+import de.tum.in.www1.artemis.web.rest.SubmissionResource;
+import de.tum.in.www1.artemis.web.rest.errors.AccessForbiddenException;
+import de.tum.in.www1.artemis.web.rest.errors.BadRequestAlertException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.ZonedDateTime;
+import java.util.Optional;
+
+/**
+ * REST controller for managing complaints.
```

Comment:
This is a service, not a rest controller

Reply:
You are right, I updated it.